src/styles/variables/tokens.scss

```diff
@@ -0,0 +1,2 @@
+@import '~@palmetto/palmetto-design-tokens/build/css/variables-color.css';
```

Comment:
Should token be a css or scss file? In this case, seems like it should be css since it contains exclusively css variables. In addition, it doesn't do any Sassy things.

Reply:
yes I intend to actually rename all these because we're not using sass features anywhere, however, the storybook webpack config complains at the moment when we do this. Needs some tweaking. Perhaps the next item to tackle as a group is improving dev/build process a bit.